test/NNODE2_tests.jl

```diff
@@ -0,0 +1,74 @@
+using Test, Flux
+using Plots
+using DiffEqDevTools
+using DiffEqBase, NeuralNetDiffEq
+
+# Homogeneous
+f(du, u, p, t) = -du+6*u
+tspan = (0.0f0, 1.0f0)
+u0 = [1.0f0]
+du0 = [0.0f0]
+dt = 1/5f0
+function an_sol(du0, u0, p, t)
+  [1/5 * exp(-3*t) *(3*exp(5*t)+2.0)]
+end
+prob = SecondOrderODEProblem(ODEFunction{false}(f, analytic=an_sol), u0, du0, tspan)
+chain = Chain(Dense(1,5,σ),Dense(5,1))
+opt = ADAM(1e-04, (0.9, 0.95))
+sol = solve(prob, NeuralNetDiffEq.NNODE2(chain,opt), dt=dt, verbose = true, maxiters = 200)
+#println(sol)
+err = sol.errors[:12]
```

Comment:
`12` -> `l2` but now `sol.errors` is empty

Reply:
Fixed typo, have not been able to get it working for 2nd order yet

Reply:
Fixed typo, haven't been able to get it to work with 2nd order yet